### PR TITLE
blog post for deprecating &$references

### PR DESCRIPTION
**Pull request by jjergus**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -81,3 +81,6 @@ kmeht:
 jjergus:
   full_name: Ján
   fbid: 100001052236684
+
+daniel:
+  full_name: Daniel
```
Comment (fredemmott) [inline _data/authors.yml]: fbid might actually be needed

Comment (jjergus) [reply _data/authors.yml]: As far as I can tell there's only 1 place it's used (picture on each blog post) and it's optional there.

I messaged Daniel to ask what name/FBID he wants, I'll update this before pushing.

Comment (dneiter) [reply _data/authors.yml]: Let's just put my full name and fbid in here:

```
dneiter: 
  full_name: Daniel Neiter
  fbid: 1405460721 
```

**Comment by jjergus** [comment]
I forgot to mention builtins and `hhast-migrate --ref-to-inout`, so added a paragraph about that.

Planning to push this tomorrow morning.

**Pull request merged**